core/src/ldml/C7043_ldml.md

```diff
@@ -27,10 +27,7 @@ Draft spec PR: <https://github.com/unicode-org/cldr/pull/1847>
 
 - Data is divided into several sections. The very first section is the `'sect'`
   section which is the table of contents.
-- All sections, including the first, begin on a 128-bit boundary with zero
-  padding as needed.
-- All variable length sections have the variable part begin on a 128-bit
-  boundary, with zero padding as needed.
+- All sections, including the first, are padded to a 32-bit boundary.
```

Comment:
There's no more padding, right?

Reply:
> There's no more padding, right?

Right. It should say aligned.

Reply:
> There's no more padding, right?

There's no padding but everything is on a 32 bit boundary